provisions/roles/registry/tasks/main.yml

```diff
@@ -12,3 +12,16 @@
   tags:
     - registry
     - jenkins/slave
+
+- name: Run container for UI for registry.
+  docker_container:
+      image: konradkleine/docker-registry-frontend:v2
```

Comment:
We will once i have the centos container up, unless we are ok building the Debian one.